src/EntityDetails/EntityTableCell.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+
+/**
+ * @param {string} value string represents table data cell value from Cell object property
+ * @returns {JSX} renders a custom table data cell
+ */
+const EntityTableCell = ({ value: initialStateValue }) => {
+  /**
+   * 1) value will be data from props you get from Cell object property
+   * 2) currentStateValue is a editable value data to display
+   * 3) isDivHidden is a boolean to check whether div is hidden or not
+   * 4) saveChanges is a boolean to check whether changes are saved
+   * 5) reset is a string to display "reset"
+   * 6) edited is a string to display "edited"
+   * */
+  const [value, setValue] = useState(initialStateValue)
+  const [currentStateValue, setCurrentStateValue] = useState(value)
+  const [isDivHidden, setIsDivHidden] = useState(true)
+  const [saveChanges, setSaveChanges] = useState(false)
+  const [reset, setReset] = useState("")
+  const [edited, setEdited] = useState("")
+
+  // Text input can be typed in the input tag, when keyboard event is trigger
+  const handleInputChange = (e) => {
+    setValue(e.target.value)
+  }
+
+  // Saves the text input, displays current state edited text input, and hide rest of the identifier tags
+  /// (e.g. button, span, etc...) when the save button triggers
+  const handleSaveChange = (e) => {
+    e.stopPropagation()
+    setSaveChanges(true)
+    setCurrentStateValue(value)
+    setIsDivHidden(true)
+    setReset("Reset")
+    setEdited("-Edited")
+  }
+
+  // Hides all identifier tags (e.g. button, div, span) when cancel button triggers
+  const handleCancelChange = (e) => {
+    e.stopPropagation()
+    setIsDivHidden(true)
+  }
+
+  // Unhides all identifier tags (e.g. button, div, span) when particular cell div triggers
+  const handleDivChange = () => {
+    setIsDivHidden(false)
+  }
+
+  // Reset current state edited text input, hides all identifier tags, and display initial state value
+  const handleResetChange = (e) => {
+    e.stopPropagation()
+    setCurrentStateValue("")
+    setValue(initialStateValue)
+    setIsDivHidden(true)
+    setSaveChanges(false)
+    setReset("")
+    setEdited("")
+  }
+
+  // Display the initial state value
+  const displayInitialStateValue = () => {
+    if (!saveChanges) {
+      return <div className="initial-state">{initialStateValue}</div>
```

Comment:
you can just create a variable about the className since everything else is the same in the jsx.

Reply:
The reason why I have two classname is because I wanted to add font size.
 `  .modified-initial-state {
    font-size: 0.75rem;
  }`

Reply:
Attached a variable to className. The method return two different string regarding above className.